test/v1_0_2/non_templating.js

```diff
@@ -3045,48 +3024,20 @@
             voiding = voiding.statement;
             voiding.id = voidingId;
             voiding.object.id = voidedId;
-            voiding.timestamp = '2015-01-01T19:09:13.245Z';
 
             request(helper.getEndpoint())
                 .post(helper.getEndpointStatements())
                 .headers(helper.addAllHeaders({}))
                 .json(voiding)
-                .expect(200, done);
```

Comment:
Only returns statement references from object.
https://github.com/adlnet/xAPI-Spec/blob/master/xAPI.md#filter-conditions-for-statementrefs